exercise10_beyond2.py

```diff
@@ -0,0 +1,17 @@
+sequence = (10, 20, 'a', '30', 'bcd')
+
+def mysum_bigger_than(*args):
+    initial_sum = 0
+    for item in sequence[0:]:
+        if type(item) == str:
+            if item.isnumeric() == True:
+                initial_sum += int(item)
+            else:
+                pass
```

Comment:
I can see you are a big fan of `pass` instruction :) Try to think if you need it in this case